test/index.html

```diff
@@ -16,22 +16,16 @@
   <script src="/node_modules/video.js/dist/video.js"></script>
   <script src="/node_modules/videojs-contrib-media-sources/src/videojs-media-sources.js"></script>
 
-  <script src="/src/videojs-hls.js"></script>
+  <script src="/src/videojs-contrib-hls.js"></script>
   <script src="/src/xhr.js"></script>
-  <script src="/src/stream.js"></script>
-  <script src="/src/m3u8/m3u8-parser.js"></script>
+  <script src="/dist/videojs-contrib-hls.js"></script>
```

Comment:
This mix of src and dist dependencies is surprising to me. Is that because this is only one step of the migration?